config/ember-try.js

```diff
@@ -1,47 +1,45 @@
 /*jshint node:true*/
 module.exports = {
+  useYarn: true,
   scenarios: [
     {
-      name: 'ember-1.12',
+      name: 'ember-lts-2.4',
```

Comment:
Why to drop 1.12. I think we definitely support it

Reply:
I'm almost sure the addon still can be installed with ember-1.12, but its really hard to make our addons tests compatible with those old versions.

ember 1.12 is more than two years old which is a long time in js terms, I think we did a good job keeping it for so long, but it's time to move on 😃 .

Unless someone feels too strong about it, I think its best to start doing some changes in the code to make it easier to maintain. That's why I'm removing a whole blueprint in this same PR.

Reply:
Yeah this blueprint looked massive. I hope nobody actively use a `component-test` blueprint 😆

Reply:
Totally, it's a candidate to be its own addon.

https://github.com/san650/ember-cli-page-object/issues/319#issuecomment-345362150